commands/label_errors.py

```diff
@@ -115,6 +115,7 @@ def main(
         ml_task: MLTaskT = job.get("mlTask")  # type: ignore
         tools: List[ToolT] = job.get("tools")
 
+        # We should delete model_framework
```

Comment:
it seems to be done below. And why?

Reply:
This flag is misleading. We cannot currently use TensorFlow with AutoML

Reply:
ok, this needs to be discussed outside of this PR since this is a change in the AutoML requirements